### ISSUE TITLE
"Which experiments" notification shown in wrong locale

### ISSUE THREAD

**Issue by IByte** [mozilla/testpilot#1872]
The "Which experiments" notification bar of the Test Pilot add-on was just shown to me in German, even though the locale of my Firefox installation is en-GB and geolocation would show Dutch.

**Comment by flodolo** [comment]
Location is not used to determine the language, browser's settings should. Having said that, en-GB is not available as a localization, so it should fallback to en-US.

Is the rest of the add-on correctly in English?

**Comment by IByte** [comment]
Actually, I just checked, and no, it isn't. If I click the Test Pilot toolbar button, the experiment names are in English, but the button below them says "Alle Experimente ansehen" (View all experiments). If I click it, I go to the Test Pilot page in English. None of the software on my system is set to German, so imagine my surprise (luckily I do speak some German, or I wouldn't have known what hit me).

(On a possibly unrelated note, now I can't open the Add-ons Manager... Firefox hangs when I try it, it goes to "Not responding" in Task Manager. By the way, I have dozens of add-ons installed, quite a lot more than average, it would seem.)

Screenshot:

![2016-11-26 13_52_38-mozilla firefox 50 0](https://cloud.githubusercontent.com/assets/708282/20640539/b0406532-b3e0-11e6-91a4-286358fad05d.png)

**Comment by pdehaan** [comment]
IIRC, the experiment names aren't translated, so those should always be English.

If you go to about:preferences#content and click on the [Choose...] button next to Languages, what does your list of languages look like?

![preferences](https://cloud.githubusercontent.com/assets/557895/20733436/3c2f710a-b648-11e6-9bb1-fd48f829ae6f.png)

**Comment by flodolo** [comment]
Even easier: open about:config in Firefox and copy/paste here the value of `intl.accept_languages` (search for it, right click, copy value).

en-GB default is `en-GB, en`

**Comment by IByte** [comment]
My intl.accept_languages is set to "en-GB,en,en-nz,en-us,nl,de-de,de,fr-fr,fr,cs".

**Comment by flodolo** [comment]
Uhm, now that I think a bit more, we're talking about an add-on, so it would be determined by `general.useragent.locale`, not the accept language (that's for the website).

**Comment by IByte** [comment]
general.useragent.locale is en-GB

**Comment by johngruen** [comment]
@6a68 will take a look at this. Let's figure out how @IByte can check build locale.

**Comment by SoftVision-PaulOiegas** [comment]
The toolbar button doorhanger content is not influenced by Content > Language option since it's part of the browser UI and not of the website. based on this I incline to think that this more of a browser issue than a Test Pilot one. 

Also, speaking of browser issues, something similar happened to me. I was testing Test Pilot locales with Content > Language option by adding multiple languages. After changing several content languages and multiple pages refreshes I got stuck on one of them. 
Even if I deleted all the languages and added only en-US, the same foreign language was still displayed on page content. 
The strangest part that this was happening even after browser restart and new profile. The only thing that made it work again was opening another version of the browser and then reopen the one with the issue.
